package.json

```diff
@@ -28,15 +28,12 @@
     "geojson-rewind": "^0.3.0",
     "geojson-vt": "^3.1.4",
     "gl-matrix": "^2.6.1",
-    "gray-matter": "^3.0.8",
     "grid-index": "^1.0.0",
     "minimist": "0.0.8",
     "pbf": "^3.0.5",
     "quickselect": "^1.0.0",
     "rw": "^1.3.3",
-    "shuffle-seed": "^1.1.6",
```

Comment:
Integration tests have their own set of dependencies: https://github.com/mapbox/mapbox-gl-js/blob/master/test/integration/package.json#L17, and this one is a duplicate. Can we remove it, or is there a gotcha with it?

Reply:
Okay, should be fine then. There is [a gotcha](https://github.com/mapbox/mapbox-gl-js/commit/8cf6a99be2f15f146f8539a0d66c4eb3f786a05b), but it doesn't apply to this case because `@mapbox/mapbox-gl-test-suite` is a dev dependency.